### PR TITLE
Add relax-and-recover to e8 profiles

### PR DESCRIPTION
**Pull request by shaneboulden**
The ACSC "Essential Eight" requires that daily backups are performed. This PR adds support for the [relax-and-recover ](https://access.redhat.com/solutions/2115051) utility to the e8 profiles, supporting backup and DR.

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Profile e8 on rhel7:
 Rule package_rear_installed added to e8 profile.
Profile e8 on rhel8:
 Rule package_rear_installed added to e8 profile.
Profile ism_o on rhel8:
 ISM_O profile extends changed E8 profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml e8
 build_product rhel8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml ism_o
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml e8

**Comment by shaneboulden** [comment]
It looks like this is failing the `stable-profiles` test for all configurations:
```
80/237 Test  #81: stable-profiles ...........................................................***Failed    0.14 sec
Following selections were added to the rhel7's e8 profile:
 - package_rear_installed
Following selections were added to the rhel8's e8 profile:
 - package_rear_installed
If changes to mentioned profiles are intentional, copy those compiled files, so they become the new reference:
cp '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel8/build/rhel7/profiles/e8.profile' '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel8/tests/data/profile_stability/rhel7/e8.profile'
cp '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel8/build/rhel8/profiles/e8.profile' '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel8/tests/data/profile_stability/rhel8/e8.profile'
Please remember that if you change a profile that is extended by other profiles, changes propagate to derived profiles. If those changes are unwanted, you have to supress them using explicit selections or !unselections in derived profiles.
```
Any guidance how I can resolve this?

**Comment by vojtapolasek** [comment]
Yes, this check is here to prevent accidental changes in stable profiles, E8 is one of them. Please also make changes in respective .profile files in tests/data/profile_stability/{product}.

**Comment by vojtapolasek** [comment]
Also please add CCE identifiers. Look into any rule.yml for the "identifiers" section.
Take identifiers from  'shared/references/cce-redhat-avail.txt' - the number without the 'CCE-' preposition. Put there please identifiers for both rhel7 and rhel8.
you can take  'linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/rule.yml' as an example.

**Comment by mildas** [comment]
**Changes identified:**
Profile e8 on rhel7:
 Rule package_rear_installed added to e8 profile.
Profile e8 on rhel8:
 Rule package_rear_installed added to e8 profile.
Profile ism_o on rhel8:
 ISM_O profile extends changed E8 profile.
Profile e8 on tests:
 Rule package_rear_installed added to e8 profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml e8
 build_product tests
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-tests-ds.xml e8
 build_product rhel8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml ism_o
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml e8

**Comment by mildas** [comment]
**Changes identified:**
Profile e8 on rhel7:
 Rule package_rear_installed added to e8 profile.
Profile e8 on rhel8:
 Rule package_rear_installed added to e8 profile.
Profile ism_o on rhel8:
 ISM_O profile extends changed E8 profile.
Profile e8 on tests:
 Rule package_rear_installed added to e8 profile.

**Recommended tests to execute:**
 build_product tests
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-tests-ds.xml e8
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml e8
 build_product rhel8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml ism_o
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml e8

File: linux_os/guide/system/software/system-tools/package_rear_installed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8
+
+title: 'Install rear Package'
+
+description: |-
+    {{{ describe_package_install(package="rear") }}}
+rationale: |-
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/system-tools/package_rear_installed/rule.yml:9]: Sory for nitpick, but for better readability please put new line before rationale.

File: linux_os/guide/system/software/system-tools/package_rear_installed/rule.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <tt>rear</tt> contains the Relax-and-Recover (ReaR) utility. ReaR produces a bootable
+    image of a system and restores from backup using this image.
+severity: medium
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/system-tools/package_rear_installed/rule.yml:12]: Also please separate rationale and severity with a new line.

**Comment by mildas** [comment]
**Changes identified:**
Profile e8 on rhel7:
 Rule package_rear_installed added to e8 profile.
Profile e8 on rhel8:
 Rule package_rear_installed added to e8 profile.
Profile ism_o on rhel8:
 ISM_O profile extends changed E8 profile.
Profile e8 on tests:
 Rule package_rear_installed added to e8 profile.

**Recommended tests to execute:**
 build_product tests
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-tests-ds.xml e8
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml e8
 build_product rhel8
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml ism_o
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel8-ds.xml e8

**Comment by vojtapolasek** [comment]
Thank you very much, merging.

**Pull request merged**